tests/baselines/reference/keyofAndIndexedAccessErrors.errors.txt

```diff
@@ -46,7 +46,6 @@ tests/cases/conformance/types/keyof/keyofAndIndexedAccessErrors.ts(87,5): error
 tests/cases/conformance/types/keyof/keyofAndIndexedAccessErrors.ts(103,9): error TS2322: Type 'Extract<keyof T, string>' is not assignable to type 'K'.
   Type 'string & keyof T' is not assignable to type 'K'.
     Type 'string' is not assignable to type 'K'.
-      Type 'string' is not assignable to type 'K'.

```

Comment:
I'm not complaining, but why'd this go away?

Reply:
I accidentally fixed a bug where we duplicated the elaboration - this is as the same as the line above.